src/commands/run-build.yml

```diff
@@ -29,6 +36,7 @@ steps:
       name: Run MATLAB build
       environment:
         PARAM_TASKS: <<parameters.tasks>>
+        PARAM_BUILD_OPTIONS: <<parameters.build-options>>
         PARAM_ARGS: <<parameters.startup-options>>
```

Comment:
Should this be something like PARAMS_STARTUP_OPTIONS now?

Reply:
Sure, refactoring it!